app/js/components/collection_widget.test.ts

```diff
@@ -1,6 +1,7 @@
 import { loadEntityOidcForm } from "./collection_widget";
 
 import * as $ from "jquery";
+import * as jQuery from "jquery";
```

Comment:
Have you considered replacing the `$` usage with `jQuery`? By the way, I have zero tension with your solution but was triggered by the duplicate import under different aliases. Leaving it like this, especially because this is a test, if perfectly fine with me.

By the way, shouldn't the Encore `autoProvidejQuery` option not provide jQuery capabilities in this script in the first place?

Reply:
Yes I've considered that, but I chose to use the `$` only for selectors and `jQuery` to use for objects in the library itself, in this case for a jQuery Event (`jQuery.Event`).